components/TextFields/src/ContainedInputView/MDCBaseTextField.m

```diff
@@ -226,6 +226,19 @@ - (CGRect)clearButtonRectForBounds:(CGRect)bounds {
   return self.layout.clearButtonFrameNormal;
 }
 
+- (CGRect)placeholderRectForBounds:(CGRect)bounds {
```

Comment:
Please add a comment to the apple docs so that our future selves know that this is an override of an app API
https://developer.apple.com/documentation/uikit/uitextfield/1619615-placeholderrectforbounds?language=objc

Reply:
This should have been under the `UITextField Layout Overrides` pragma but wasn't. I just put it there 👍